modules/entity/src/models.ts

```diff
@@ -1,40 +1,40 @@
-export type ComparerStr<T> = {
+export interface ComparerStr<T> {
   (a: T, b: T): string;
-};
+}
```

Comment:
I think even simpler is to write this:

type ComparerStr<T> = (a: T, b: T) => string;

Here and below for the other function types.

(tsickle chokes on the old code, but I think the new code won't help there anyway, so you might as well write it in the simplest format, which tsickle also won't choke on.)

Reply:
It takes new code, I verified it. But, yeah, for function types it looks even simpler with `type`